src/main/java/fr/xephi/authme/process/join/AsynchronousJoin.java

```diff
@@ -179,8 +179,9 @@ private void processJoinSync(Player player, boolean isAuthAvailable, Location lo
 
     private boolean canResumeSession(Player player) {
         final String name = player.getName();
-        if (database.isLogged(name)) {
+        if (database.hasSession(name)) {
             database.setUnlogged(name);
+            database.revokeSession(name);
             playerCache.removePlayer(name);
             if(service.getProperty(PluginSettings.SESSIONS_ENABLED)) {
```

Comment:
`playerCache.removePlayer(name);` necessary? A player should never be in the PlayerCache at this point in time.

Reply:
Probably some code left there from the previous bungeecord cache implementation